ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml

```diff
@@ -18,12 +18,53 @@ periodics:
     labels:
       master: ci.openshift.redhat.com
     name: azure_build_node_image_centos
-- agent: jenkins
+- agent: kubernetes
   cron: ""
+  decorate: true
   interval: 24h
   labels:
     master: ci.openshift.redhat.com
   name: azure_build_base_image_rhel
+  skip_cloning: true
+  spec:
+    containers:
+    - args:
+      - --artifact-dir=$(ARTIFACTS)
+      - --secret-dir=/usr/local/e2e-azure-secret
+      - --template=/usr/local/build-base-image
+      - --target=build-base-image
+      command:
+      - ci-operator
+      env:
+      - name: CONFIG_SPEC
+        valueFrom:
+          configMapKeyRef:
+            key: release-3.10.yaml
+            name: ci-operator-openshift-openshift-ansible
+      - name: JOB_NAME_SAFE
+        value: build-base-image
+      - name: CLUSTER_TYPE
+        value: azure
+      image: ci-operator:latest
+      imagePullPolicy: Always
+      name: ""
+      resources: {}
+      volumeMounts:
+      - mountPath: /usr/local/build-base-image
+        name: job-definition
+        subPath: build-base-image.yaml
+      - mountPath: /usr/local/e2e-azure-secret
```

Comment:
according to comments in github.com/openshift/release/pull/1774 i might need to add a suffix like -base-image to avoid surprises when pods of other jobs are configured with the same --secret-dir values. WDYT?

Reply:
same comment as above.